fw/control.c

```diff
@@ -53,38 +54,50 @@ int8_t control_run_message(control_message* m) //{{{
 	{
 		case CONTROL_TYPE_INIT:
```

Comment:
Double check to make sure this change did not break reset after trace functionality.